includes/polling/functions.inc.php

```diff
@@ -56,7 +56,7 @@ function poll_sensor($device, $class)
     $misc_sensors = array();
     $all_sensors = array();
 
-    foreach (dbFetchRows("SELECT * FROM `sensors` WHERE `sensor_class` = '?' AND `device_id` = ?", array($class, $device['device_id'])) as $sensor) {
+    foreach (dbFetchRows("SELECT * FROM `sensors` WHERE `sensor_class` = ? AND `device_id` = ?", array($class, $device['device_id'])) as $sensor) {
```

Comment:
Ditto and we don't all over the show but for some reason people have been having issues recently and no one's been able to track it down. So unless this is causing you problems then we should leave it for now.